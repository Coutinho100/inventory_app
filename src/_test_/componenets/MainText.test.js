import React from 'react';
import MainText from '../../components/Component/MainText';
import { Text } from 'react-native';
import { configure  } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import test_function from './test_function';


configure({ adapter: new Adapter() });

test_function(["<Button View/>",1],['renders correctly','renders the length of view'],[<MainText />,Text],null)
