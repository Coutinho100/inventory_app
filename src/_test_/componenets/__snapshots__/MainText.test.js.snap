// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button View/> renders correctly 1`] = `
<Text
  style={
    Object {
      "backgroundColor": "transparent",
      "color": "#29aaf4",
    }
  }
/>
`;
